### ISSUE TITLE
Instruction regarding libsodium

### ISSUE THREAD

**Issue by psychomb** [cardano-foundation/stake-pool-school-handbook#26]
## The instructions for the IOHK-fork install of libsodium might benefit from the following minor change.

In  *stake-pool-school-handbook/stake-pool-guide/getting-started/install-node.md*, my experience shows that the export PATHs should be performed before the installation of libsodium rather than after.

````
export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
export PKG_CONFIG_PATH="/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH"

git clone https://github.com/input-output-hk/libsodium
cd libsodium
git checkout 66f017f1
./autogen.sh
./configure
make
sudo make install
````

**Comment by psychomb** [comment]
Now included in PR #28

**Comment by rdlrt** [comment]
Can be closed?
